lms/urls.py

```diff
@@ -99,6 +99,9 @@
     # Entitlement API RESTful endpoints
     url(r'^api/entitlements/', include('entitlements.api.urls', namespace='entitlements_api')),
 
+    # Edly Panel API RESTful endpoints
+    url(r'^api/monthly_active_users/', include('openedx.features.edly_panel.api.urls', namespace='edly_panel_api')),
```

Comment:
API and its UR for `monthly_active_users` should be inside the `edly_panel` app.

Here we can use something like:
`api/eldy_panel/`
and inside the app we can have the final url
`api/eldy_panel/v1/monthly_active_users/`

Reply:
done.